docs/sharding_representation.md

```diff
@@ -101,8 +101,8 @@ An open dimension is open for propagation to further shard it along additional
 axes, i.e. the specified dimension sharding doesn't have to be the final
 sharding of that dimension. This is similar (but not exactly the same as) to
 
-*   [`jax.sharding.PartitionSpec.UNCONSTRAINED`](https://jax.readthedocs.io/en/latest/jax.sharding.html#jax.sharding.PartitionSpec)
-*   GSPMD's `unspecified_dims`
+* [`jax.sharding.PartitionSpec.UNCONSTRAINED`](https://jax.readthedocs.io/en/latest/jax.sharding.html#jax.sharding.PartitionSpec)
+* GSPMD's `unspecified_dims`
```

Comment:
Oops my mistake I missed this: kinda like in a previous review, since UNCONSTRAINED isn't in the public documentation, let's exclude it for now.

Reply:
Let me know if this is what you had in mind.

Reply:
Ya this looks good!